utopia-remix/jest.config.js

```diff
@@ -3,4 +3,5 @@ export default {
   transform: {
     '\\.[jt]sx?$': 'babel-jest',
   },
+  transformIgnorePatterns: ['node_modules/(?!@ngrx|(?!deck.gl)|ng-dynamic)'],
```

Comment:
Huh?

Reply:
I had to do this because it was giving jest was having a stroke due to some imports in `url-join` 😬

Reply:
I'd throw a comment in there about that then, otherwise we'll never know in 10 months time when coming back to this.

Reply:
[`c574a18` (#4923)](https://github.com/concrete-utopia/utopia/pull/4923/commits/c574a181ba3204ea8bf11b5c4d890bb0f5719922)